File: Actividad 2/Promedio.c
#include <stdio.h>

int main() {
    int i = 0;
    int num =0;
    int sum =0;
    float promedio;

    printf("Ingrese 10 numeros enteros:\n");

    for (i = 0; i < 10; i++) {
        printf("Valor %d: ", i + 1);
        if (scanf("%d", &num) != 1) {
            printf("Ingrese solo numeros enteros.\n");
            return 1;
        }
        sum += num;
    }

    promedio = sum / 10.0;
    printf("El valor de la sumatoria es: %d\n", sum);
    printf("El promedio es: %.2f\n", promedio);

    return 0;
}

File: Actividad 2/Promedio.py
print("Ingrese 10 numeros enteros")
try:
    total_sum = 0
    for i in range(10):
        num = int(input(f"Valor {i+1}: "))
        total_sum += num
    print(f"El valor de la sumatoria es: {total_sum}")
    print(f"El promedio es: {total_sum / 10}")

except ValueError:
    print("Ingrese solo números enteros.")
